test/inheritAttributes.test.js

```diff
@@ -9,6 +9,7 @@ import { stringToMpdXml } from '../src/stringToMpdXml';
 import errors from '../src/errors';
 import QUnit from 'qunit';
 import { toPlaylists } from '../src/toPlaylists';
+import decodeB64ToUint8Array from '@videojs/vhs-utils/cjs/decode-b64-to-uint8-array';
```

Comment:
in inheritAttributes, we import the `es` version, it would be better to stay consistent. https://github.com/videojs/mpd-parser/blob/fb0d2980c05d3c582ea17282ef2b2446f29973e7/src/inheritAttributes.js#L7
```suggestion
import decodeB64ToUint8Array from '@videojs/vhs-utils/es/decode-b64-to-uint8-array';
```

Reply:
Suggestion applied